src/App.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setSearchValue] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState(moviesFromServer);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const inputValue = event.target.value;
+
+    setSearchValue(inputValue);
+
+    const filteredMovies = moviesFromServer.filter((movie) => {
+      return (
+        movie.title.toLowerCase()
+          .split(' ').join('')
+          .includes(inputValue.toLowerCase().split(' ').join(''))
+        || movie.description.toLowerCase()
+          .split(' ').join('')
+          .includes(inputValue.toLowerCase().split(' ').join(''))
```

Comment:
checklist p2
try to move the same logic to helper functions

Reply:
fixed, thank you